monero-outreach-docs/translations/pt-br/storyboard/fundamental_pt_br.md

```diff
@@ -0,0 +1,79 @@
+# Preservando o direito fundamental à privacidade
+
+Jack M | 28/09/18 Monero pode ser utilizado por qualquer pessoa, independentemente de sua raça, gênero, preferência sexual ou religiosa.
+
+**"Liberdade significa a supremacia dos direitos humanos em qualquer lugar. Nosso apoio vai para todos aqueles que têm dificuldade em conquistar e manter esses direitos. Nossa força é nossa unidade de propósito. Para este alto conceito, não pode haver outro fim senão a vitória."**
+Franklin D. Roosevelt
+As Quatro Liberdades, 6 de Janeiro de 1941
+
+O fim da Segunda Guerra Mundial marcou o que hoje vem a ser a maior vitória para a liberdade no mundo. Os que defendiam a liberdade não poderiam mais ser condescendente, já que o ditado "a liberdade não vem de graça" foi cimentado de vez na mente de todos. Hoje, as mesmas liberdades básicas estão sob ataque pesado.
+
+Embora o meio de opressão tenha mudado para o cenário digital, os mesmos princípios de liberdade permanecem. As liberdades delineadas na Declaração Universal dos Direitos Humanos, escritas pelas Nações Unidas em 1948, são as mesmas que estão sendo atacadas em nosso mundo cada vez mais digitalizado. Os truques dos violadores dos direitos humanos permanecem os mesmos.
+
+Vamos dar uma olhada em um preceito na declaração: Artigo 12
+
+_Ninguém sofrerá intromissões arbitrárias na sua vida privada, na sua família, no seu domicílio ou na sua correspondência, nem ataques à sua honra e reputação. Contra tais intromissões ou ataques todas as pessoas têm direito a proteção da lei._
+
+O direito à privacidade é importante para a liberdade porque
+
+* A vigilância constante corrói o princípio de presunção de inocência
+* A vigilância constante reduz a liberdade de expressão, especialmente nas formas digitais
+* A privacidade ajuda os direitos dos cidadãos à liberdade de movimento e de associação
+
+É aqui que o Monero se destaca. Privacidade no seu uso do dinheiro constitui um elemento essencial do Artigo 12. Como você poderia expressar suas liberdades ao doar para causas nobres nas quais acredita se estiver preocupado com o fato dessas causas não se encaixarem com a narrativa de uma autoridade?
```

Comment:
Suggestion: "como vc poderia expressar suas liberdades doando para causas nobres" I think it will make the text more fluent.